ci/matrix.xml

```diff
@@ -132,7 +151,11 @@
         'set scalaJSOptimizerOptions in testSuite ~= (_.withDisableOptimizer(true))' \
         'set scalaJSStage in Global := FastOptStage' \
         ++$scala testSuite/test \
-        testSuite/clean &&
+        testSuite/clean
```

Comment:
Indeed. We need `clean` in between invocations of the linker when the options change. We don't a `clean` at the end of a job. That said, it doesn't bother me to leave it there, for uniformity.